src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -120,62 +157,428 @@ def _load_iges(self):
+                -3.18000000e-03],
+               [ 5.00000000e+00,  0.00000000e+00, -7.62000000e-03,
+                 0.00000000e+00],
+               [ 6.00000000e+00,  1.58750000e-02, -7.62000000e-03,
+        ...
+
+        """
+        if return_as_array or return_ids_in_array:
+            if return_ids_in_array:
+                keyp = np.array(self._keypoints[0])
+                keyp_num = np.array(self._keypoints[1])
+                return np.hstack((keyp_num.reshape((-1, 1)), keyp))
+            else:
+                return np.array(self._keypoints[0])
+
+        keypoints, kp_num = self._keypoints
+
+        if return_as_list:
+            keypoints_ = []
+
+            for each_point, each_id in zip(keypoints, kp_num):
+                keypoints_ps = pv.PolyData([each_point])
+                keypoints_ps["entity_num"] = np.array(each_id).reshape((1))
+                keypoints_.append(keypoints_ps)
+
+            return keypoints_
+
+        else:
+            keypoints_pd = pv.PolyData(keypoints)
+            keypoints_pd["entity_num"] = kp_num
+            return keypoints_pd
 
     @property
-    def _lines(self):
+    def _lines(self) -> List[pv.PolyData]:
         """Returns lines cache"""
         if self._lines_cache is None:
             self._lines_cache = self._load_lines()
         return self._lines_cache
 
     @property
-    def lines(self):
-        """Active lines as a pyvista.PolyData"""
-        return self._lines
+    def lines(self) -> pv.MultiBlock:
+        """Obtain the lines geometry
 
-    def areas(self, quality=1, merge=False):
-        """List of areas from MAPDL represented as ``pyvista.PolyData``.
+        Obtain the selected lines as a :class:`pyvista.MultiBlock` object.
+
+        Returns
+        -------
+        pv.MultiBlock
+
+        Examples
+        --------
+
+        >>> mapdl.geometry.lines
+        MultiBlock (0x147b77e20)
+          N Blocks    45
+          X Bounds    -0.016, 0.016
+          Y Bounds    -0.008, 0.018
+          Z Bounds    -0.003, 0.015
+
+        You can plot the entities:
+
+        >>> mapdl.geometry.lines.plot()
+
+        You can access the individual lines using indexing:
+
+        >>> line0 = mapdl.geometry.lines[0]
+        >>> line0
+        PolyData (0x147d5b220)
+          N Cells:    1
+          N Points:   100
+          N Strips:   0
+          X Bounds:   0.000e+00, 0.000e+00
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, -3.180e-03
+          N Arrays:   1
+
+        Or using the entity name:
+
+        >>> line1 = mapdl.geometry.lines["line 1"]
+        >>> line1
+        PolyData (0x147d5b220)
+          N Cells:    1
+          N Points:   100
+          N Strips:   0
+          X Bounds:   0.000e+00, 0.000e+00
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, -3.180e-03
+          N Arrays:   1
+
+        You can use this to iterate over the different elements:
```

Comment:
Again, the "this" has no clear antecedent. You can use what? ... the preceding code?

Reply:
Got it.